### DISCUSSION TITLE
Olimex ESP-32 ISO mit POE zu heiss???

### DISCUSSION THREAD

**Discussion by Parateam** [tbnobody/OpenDTU#466] [Q&A]
Hallo, ich habe gerade ein POE-ISO von Olimex geliefert bekommen. 

Das Board funktioniert soweit, habe es mit VS-Code geflasht und läuft mit POE-Ethernet. Es ist noch kein Funkmodul angeschlossen - also keine weiteren Verbraucher. Angeschlossen an einen Unify Switch 48 POE Pro. Der zeigt mir 1.8W Verbrauch am Port an.

Auf der Rückseite gibt es mittig in der Nähe der RJ45 Buchse einen kleine schwarzen Chip den man nicht länger als 2s mit dem Finger berühren kann. Er wir gleich nach dem Anstecken warm und bleibt auf der Temperatur. Ich habe den Eindruck, dass sich eine Plastikfolie darunter wellig wird.

Ist das normal? Ich habe kein gutes Gefühl, auch für die Haltbarkeit.

LG Peter

**Comment by Solarteur** [comment]
Danach
https://www.olimex.com/forum/index.php?topic=7255.0
scheint das "normal" zu sein.

**Comment by Parateam** [comment]
Ich habe das auch gelesen. Hast du das Board mal in Betrieb gehabt? Ich hätte gerne von jemanden eine Bestätigung das das bei mir im Rahmen ist. Und wie ist die Haltbarkeit wenn Halbleiter so heiss werden?

**Comment by Solarteur** [comment]
Ich habe kein solches Board. Generell sinkt die Haltbarkeit von Halbleitern mit steigender Betriebstemperatur. Ob das in diesem Fall relevant ist, bleibt "abzuwarten".

**Comment by brainpaincgn** [comment]
@Parateam, ich habe ein ähnliches Board gekauft (ESP32-POE-ISO-EA-IND). Wo hast du die Pinbelegung her, respektive könntest du mir diese schicken?
Danke im Voraus

**Reply by Parateam** [reply]
https://github.com/tbnobody/OpenDTU/discussions/429

hier gibt es eine Übersicht. Musst allerdings noch selbst compilieren VS Code weil du einen anderen Boardtyp hast. Pins und Ethernet ändert sich dann.

Schau dir doch mal die Probleme an, dich ich heute dort gepostet habe und binde dich doch ein. Du hast ja ein etwas anderes Board.
